src/framework/audio/engine/internal/engineplayer.cpp

```
@@ -31,27 +32,108 @@ using namespace muse::audio;
+    m_currentPosition = newTime;
+    m_timeChanged.send(m_currentPosition.time());
+}
+
+TimePosition EnginePlayer::proc_onTimeChanged(const TimePosition& delta)
+{
+    ONLY_AUDIO_PROC_THREAD;
+
+    // Check: Count down
+    if (!m_countDown.is_zero()) {
+        m_countDown -= delta.time();
+
```

Cppcheck (2.20.0)

[style] The function 'isValid' is never used.

(unusedFunction)